cli/dcoscli/data/help/experimental.txt

```diff
@@ -8,21 +9,23 @@ Usage:
                                   (--dcos-package=<dcos-package> |
                                     (--package-name=<package-name>
                                       [--package-version=<package-version>]))
-    dcos experimental package build [--json]
+    dcos experimental package build <build-definition>
+                                    [--json]
                                     [--output-directory=<output-directory>]
-                                    <build-definition>
-    dcos experimental service start [--json]
+    dcos experimental service start <package-name>
+                                    [--json]
                                     [--package-version=<package-version>]
                                     [--options=<options-file>]
-                                    <package-name>
 
 Commands:
     package add
         Adds a DC/OS package to DC/OS.
     package build
-        Build a package locally to install to DC/OS or share with Universe.
+        Build a package locally to add to DC/OS or share with Universe.
```

Comment:
Is this more grammatically correct? Build a package locally to be added to DC/OS or ...